netlify-templates/nextjs-toolbox
# Adds project blurb to Toolbox

## Summary

Since we aren't certain where a user might be coming from, we would like for our template to give a bit of information on what is included and where they can find more information.

In addition, I wanted to bump some Next versions

| Before | After |
| :---: | :---: |
| ![Screenshot of Next.js Toolbox without project blurb](https://user-images.githubusercontent.com/8431042/164502514-28d6721d-dbc2-4e15-8157-0a5ae36a11ba.png) | ![CleanShot 2022-04-21 at 13 24 28](https://user-images.githubusercontent.com/8431042/164516617-58940719-fd28-48a8-90e2-bdede57206cb.png) |

## Manual Testing

- Verify that all links point to the appropriate locations (e.g. if nextjs.org is the text, it should lead to https://nextjs.org)
- Verify the deploy opens to the main page

## Related Documentation or Issues

- #27 
- Prior art in Nuxt https://github.com/netlify-templates/nuxt-toolbox/issues/27
